src/lib/components/LoginForm.svelte

```
@@ -0,0 +1,154 @@
+	import Loader2 from '~icons/lucide/loader2';
+	import PwInput from './ui/origin-svelte/PwInput.svelte';
+	import autoAnimate from '@formkit/auto-animate';
+
+	interface Props {
+		formSetup: SuperValidated<
+			{
+				email: string;
+				password: string;
+				totp_code?: string | undefined;
+			},
+			any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)